### ISSUE TITLE
[Client] Update layers

### ISSUE THREAD

**Issue by indisoluble** [comodoo/meta-comodoo#24]
meta-comodoo depends on a few 3rd party layers which we have not updated recently. At some point, we should move them forward to their latest stable commits.

**Comment by indisoluble** [comment]
Also, use `OSSystems/meta-browser` instead of `mozilla-japan/meta-browser` (which is a fork of the first) given that the former already support **Firefox 45**.

**Comment by indisoluble** [comment]
Also, most of our layers are based on Krogoth, but there is now a new release: [Morty](https://www.yoctoproject.org/yoctoproject/morty-0)

**Comment by indisoluble** [comment]
On the bright side, we can build a new image with the following build configuration:
```
Build Configuration:
BB_VERSION        = "1.32.0"
BUILD_SYS         = "x86_64-linux"
NATIVELSBSTRING   = "universal"
TARGET_SYS        = "arm-poky-linux-gnueabi"
MACHINE           = "raspberrypi3"
DISTRO            = "poky"
DISTRO_VERSION    = "2.2.1"
TUNE_FEATURES     = "arm armv7ve vfp thumb neon vfpv4 callconvention-hard cortexa7"
TARGET_FPU        = "hard"
meta              
meta-poky         
meta-yocto-bsp    = "morty:e292e935b077a95ae093d4b425c7e95b177b030b"
meta-browser      = "morty:7d37ae2adec5cc5ea27367ce125bc7bce2310ca1"
meta-openstack    = "morty:6043a4a17dc5ca9080dac7789e38620b7b1aa372"
meta-comodoo      
meta-rpi-fixed-fb = "morty:0f98d2bf569a7799aa4da4d24847dda7deed2f62"
meta-openbmc      = "helium:6f113cb3a8671794e36edec9500cdf45ccadea19"
meta-gnome        
meta-oe           
meta-python       = "morty:fe5c83312de11e80b85680ef237f8acb04b4b26e"
meta-raspberrypi  = "morty:cce6292e41493158c26f5b1b7fded97faacf10d7"
meta-small-arm-extra = "morty:fa0da66cd3e598df590b0bfdfe212e1458134a15"
```

However, once I copy this image into the SD and switch on the RPi, the WIFI is not available.

I think the problem is located in `meta-raspberrypi` and it was solved with [this commit in master|http://git.yoctoproject.org/cgit.cgi/meta-raspberrypi/commit/?id=6ce6e57782c23fd21ad47286ac8b809ce989ea78], however it is not available in `morty` yet. I tried to compile `master` but it did not work and when I tried to build a new image using `krogoth` for `meta-raspberrypi` the resulting image was not able to boot the RPi.

**Comment by indisoluble** [comment]
Although we are now able to build an image based on `morty`, we have had to fork the *official repo* for `meta-raspberrypi` in order to solve [a problem with the WIFI](https://github.com/agherzan/meta-raspberrypi/commit/6ce6e57782c23fd21ad47286ac8b809ce989ea78). I am going to close this issue now but I am going to create a new one to keep an eye on the [official meta-raspberrypi/morty](https://github.com/agherzan/meta-raspberrypi/tree/morty) and use it as soon as the problem is solved.
